sourced/ml/cmd/repos2bow.py

```diff
@@ -11,21 +11,22 @@
 
 
 @pause
-def repos2bow_template(args, select=HeadFiles, cache_hook=None, save_hook=None):
+def repos2bow_template(args, select=HeadFiles, cache_hook=None, save_hook=None,
+                       full_pipeline=True):
```

Comment:
This becomes too complex. We need to split this into two functions: one can be `index_bag_features` and the second `repos2bow_template`. The second calls the first.